source/Minable.cpp

```diff
@@ -152,6 +152,9 @@ bool Minable::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 		}
 		for(const auto &it : payload)
 		{
+			if(it.second < 0)
```

Comment:
You might as well also skip payloads with a count of 0.
```suggestion
			if(it.second < 1)
```